crates/rethnet_rpc_server/build.rs

```diff
@@ -0,0 +1,35 @@
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[derive(Debug, Serialize, Deserialize)]
+struct CargoToml {
+    dependencies: HashMap<String, DependencyValue>,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(untagged)]
+enum DependencyValue {
+    String(String),
+    Object {
+        version: String,
+        git: Option<String>,
+        rev: Option<String>,
+    },
+}
+
+fn main() {
+    let cargo_toml: CargoToml = toml::from_str(include_str!("../rethnet_evm/Cargo.toml"))
```

Comment:
Does this work when you run `cargo b --all` from a different directory?

Reply:
I tried it from both the top-level directory of the repo and from a directory deep within the tree (specifically `crates/rethnet_eth/src/remote/methods`) and both worked fine.